ember-cli-build.js

```diff
@@ -110,14 +107,70 @@ module.exports = function() {
     rollupPackage(packagesES, '@ember/-internals/container'),
   ]);
 
+  let bundlesES5 = toTranspiledBundles(
+    toES5,
+    loader,
+    license,
+    nodeModule,
+    packagesESRollup,
+    dependenciesES,
+    templateCompilerDependenciesES
+  );
+
+  let bundlesIE = new Funnel(bundlesES5, { destDir: 'ie' });
+
+  let bundlesES6 = toTranspiledBundles(
+    toES6,
+    loader,
+    license,
+    nodeModule,
+    packagesESRollup,
+    dependenciesES,
+    templateCompilerDependenciesES
+  );
+
+  let emberTestsEmptyTestem = new Funnel('tests', {
+    files: ['testem.js'],
+    destDir: '',
+    annotation: 'tests/testem.js',
+  });
+
+  return new MergeTrees([
+    new Funnel(es, { destDir: 'es' }),
+    pkgAndTestESBundleDebug,
+    bundlesES6,
+    bundlesIE,
+    emberTestsEmptyTestem,
+    nodeTests(),
+
+    // test harness
+    testIndexHTML(),
+    jquery(),
+    qunit(),
+  ]);
+};
+
+function toTranspiledBundles(
```

Comment:
There's a lot going into this function. Main thing I was trying to avoid is creating the exact same trees twice, given some of them can be fairly expensive (like rollup, I believe).